bluepyopt/ephys/parameters.py

```diff
@@ -228,7 +228,7 @@ def instantiate(self, sim=None, icell=None):
 
         for location in self.locations:
             for isection in location.instantiate(sim=sim, icell=icell):
-                for seg in isection:
+                for seg in isection.allseg():
```

Comment:
Mmm, ok: http://www.neuron.yale.edu/phpbb/viewtopic.php?f=2&t=2150
And Neurodamus is applying the 'allseg' behaviour ? We need to check if that's correct